packages/amplication-data-service-generator/src/server/app-module/create-app-module.ts

```diff
@@ -66,7 +66,9 @@ export async function createAppModule(
         const playground = configService.get("GRAPHQL_PLAYGROUND");
         const introspection = configService.get("GRAPHQL_INTROSPECTION");
         return {
-          autoSchemaFile: true,
+          // @todo: allow some documented env variable to override this (e.g. GQL_SCHEMA_EXPORT_PATH)
+          autoSchemaFile: "schema.graphqls",
```

Comment:
```suggestion
          autoSchemaFile: "schema.graphql",
```
by the convention of 
- https://docs.nestjs.com/graphql/quick-start
- https://www.graphql-code-generator.com/

Reply:
as far as I understand, those are just some defaults and examples, not conventions.

fyi, some projects contain multiple services/backends and generate multiple schemas and use schema stitching, federation, etc. to combine them.

from my experience, different extensions make it easier to distinguish between _schema_ (SDL) files and _query/mutation/fragment_ files, and make it easier to work with such tools as graphql-let and graphql-codegen, especially in mono-repos.

p.s., here are some links that I was able to find in 10 seconds to other projects that use the `graphqls` extension, so you know I'm not making this up 😄 :
https://github.com/piglovesyou/graphql-let
https://devdocs.magento.com/guides/v2.4/graphql/develop/create-graphqls-file.html
https://github.com/howtographql/graphql-java/blob/master/src/main/resources/schema.graphqls